### ISSUE TITLE
[Bug Report] Deploy fine-tune model got the parameter error message

### ISSUE THREAD

**Issue by eric80116** [aws/amazon-sagemaker-examples#4438]
**Link to the notebook**
https://github.com/aws/amazon-sagemaker-examples/blob/main/introduction_to_amazon_algorithms/jumpstart-foundation-models/llama-2-finetuning.ipynb

**Describe the bug**
When deploy the fine-tune model with the following code and got the parameter error message
`finetuned_predictor = estimator.deploy()`

**To reproduce**
Just execute it step by step until deploy the fine-tune model

**Logs**
ParamValidationError: Parameter validation failed:
Unknown parameter in PrimaryContainer: "ModelDataSource", must be one of: ContainerHostname, Image, ImageConfig, Mode, ModelDataUrl, Environment, ModelPackageN
[llama-fine-tune-error.log](https://github.com/aws/amazon-sagemaker-examples/files/12644540/llama-fine-tune-error.log)
ame, InferenceSpecificationName, MultiModelConfig

**Comment by sparshdoc32** [comment]
are you using Boto to create the model?
I was following this doc -> https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-uncompressed.html
and got the same error, after updating the boto version it got fixed.
Maybe you can try that.

**Comment by Jacobsolawetz** [comment]
Same here

**Comment by tleyden** [comment]
@eric80116 did you solve it?  This issue might be relevant: https://github.com/aws/sagemaker-python-sdk/issues/4321

**Comment by eric80116** [comment]
@tleyden No, I still got the same error even I upgraded boto3 to 1.34.2 and sagemaker to 2.200.1.
